### PR TITLE
Network request

### PR DESCRIPTION
**Pull request by IvanJanuary**

### CODE REVIEW

File: PersonDataRequest/PersonDataRequest/Gallery.swift [hunk 1]
```diff
@@ -21,9 +25,9 @@ class Gallery {
     }
     
     func setup() {
-        let img1 = Picture(image: UIImage(named: "img1")!)
-        let img2 = Picture(image: UIImage(named: "img2")!)
-        let img3 = Picture(image: UIImage(named: "img3")!)
+        let img1 = Picture(image: UIImage())
+        let img2 = Picture(image: UIImage())
+        let img3 = Picture(image: UIImage())
```
Comment (Lana-Resquit) [inline PersonDataRequest/PersonDataRequest/Gallery.swift:30]: Why first 3 images were deleted and others are still in place?

File: PersonDataRequest/PersonDataRequest/ApiHelper.swift [hunk 2]
```diff
@@ -6,6 +6,7 @@
 //
 
 import Foundation
+import UIKit
```
[the remark this answers is not in the document]
Comment (IvanJanuary) [reply PersonDataRequest/PersonDataRequest/ApiHelper.swift:9]: Done

File: PersonDataRequest/PersonDataRequest/Gallery.swift [hunk 3]
```diff
@@ -21,9 +25,9 @@ class Gallery {
...
         let img4 = Picture(image: UIImage(named: "img4")!)
         let img5 = Picture(image: UIImage(named: "img5")!)
```
[the remark this answers is not in the document]
Comment (IvanJanuary) [reply PersonDataRequest/PersonDataRequest/Gallery.swift:32]: Fixed

File: PersonDataRequest/PersonDataRequest/Gallery.swift (hunk 1 above)
Comment (IvanJanuary) [reply PersonDataRequest/PersonDataRequest/Gallery.swift:30]: Fixed

**Pull request merged**